File: MODUL_4/Py/PRAK403_2310817310014_MUHAMMADRIZKISAPUTRA.py
angka1, angka2 = map(int, input("Masukkan dua angka: ").split())

waktu = 1 if angka1 < angka2 else -1

for i in range(angka1, angka2, waktu):
    print(f"{i} {angka1 + angka2 - i} - ", end='')

print(f"{angka2} {angka1}")

File: MODUL_4/C/PRAK403_2310817310014_MUHAMMADRIZKISAPUTRA.c
#include <stdio.h>

int main() {
    int angka1, angka2;

    printf("Masukkan dua angka : ");
    scanf("%d %d", &angka1, &angka2);

    int waktu = (angka1 < angka2) ? 1 : -1;

    for (int i = angka1; i != angka2; i += waktu) {
        printf("%d %d - ", i, angka1 + angka2 - i);
    }

    printf("%d %d\n", angka2, angka1);

    return 0;
}
